diff --git a/node_modules/commander/typings/index.d.ts b/node_modules/commander/typings/index.d.ts
index b2fd988..182ab0c 100644
--- a/node_modules/commander/typings/index.d.ts
+++ b/node_modules/commander/typings/index.d.ts
@@ -25,7 +25,7 @@ declare namespace local {
     constructor(flags: string, description?: string);
   }
 
-  class Command extends NodeJS.EventEmitter {
+  class Command {
     [key: string]: any;
 
     args: string[];
